docs/environment-setup.md

```diff
@@ -168,6 +168,14 @@ _Note: If experiencing an error such as `Error starting userland proxy: listen t
 
 To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
 
-### Finally
+### Finally:
 
-Run `npm start` and open `localhost:3000`
+Telescope requires a running back-end and front-end to start. The following steps assume you are running the back-end locally. For more information on running the front-end with our staging/production as the back-end, please refer to our [Front-End Document](gatsbyjs.md).
+
+Start the back-end for Telescope:
+Run `npm start`
+
```

Comment:
Add a note here that this will trigger the backend server to start downloading feeds and processing them.  This will go on forever until you quit the app.  The server is now running on `http://localhost:3000` (or whatever `PORT` you specified in the `.env`).  You can hit the endpoints manually, for example: `http://localhost:3000/posts` or `http://localhost:3000/feeds`, etc.  You need a full front-end to do anything useful with it.